src/main/java/fr/pilato/elasticsearch/crawler/fs/util/FsCrawlerUtil.java

```diff
@@ -221,6 +222,10 @@ public static LocalDateTime getCreationTime(File file) {
         return time;
     }
 
+    public static String getFileExtension(File file) throws Exception {
+ 	return FilenameUtils.getExtension(file.getAbsolutePath()).toLowerCase();
```

Comment:
Actually this part does not throw any Exception. Just remove `throws Exception` and we should be good.
Beware of spaces vs tabs. Prefer using spaces for indentation please.